crowbar_framework/app/barclamps/hadoop_barclamp.rb

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class HadoopBarclamp < Crowbar::Registry::Barclamp
+  name "hadoop"
+  display "Hadoop"
+  description "Hadoop Core component"
+
+  member [
+    "apachehadoop"
+  ]
+
+  requires [
+    "@crowbar"
+  ]
+
+  listed true
+
+  layout 1
+  version 0
+  schema 3
+
+  order 200
+
+  nav(
+    barclamps: {
+      hadoop: {
+        order: 40,
+        route: "index_barclamp_path",
+        params: {
+          controller: "hadoop"
+        }
+      }
+    },
+    help: {
+      hadoop_users: {
+        order: 50,
+        path: "/docs/hadoop_users_guide.pdf",
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.